tablo/api.py

```diff
@@ -478,6 +478,9 @@ def deploy(self, request, **kwargs):
         except InternalError as e:
             logger.exception(e)
             raise ImmediateHttpResponse(HttpBadRequest('Error deploying file to database.'))
+        except Exception as e:
```

Comment:
The follow up is same as above, and `Exception` should catch everything not caught above.  Is that the intent (are you sure you don't want a more specific exception than `Exception`)?  If so, then you could just drop lines 478-480.